src/components/Dashboard/StaffNotation.vue

```diff
@@ -66,7 +66,7 @@
             <v-list-item
               data-type="registrars-notation"
               @click="showRegistrarsNotationDialog()"
-              :disabled="disabled"
+              :disabled="!isAllowed(AllowableActions.REGISTRARS_NOTATION)"
```

Comment:
This "is allowed" stuff is part of the allowed actions change and will go away after a rebase.

Reply:
It's gone now!